perllib/FixMyStreet/Cobrand/NottinghamshirePolice.pm

```diff
@@ -25,7 +30,29 @@ use Moo;
 sub council_area_id { [ 2236, 2565 ] }
 sub council_area { 'Nottinghamshire'; }
 sub council_name { 'Nottinghamshire Police' }
-sub council_url { 'nottinghamshirepolice' }
+
+=item * Any superuser or staff user can access the admin
```

Comment:
Just appears as a shorter list of bullet points, e.g. see Lincolnshire at https://pages.mysociety.org/fixmystreet-pod/FixMyStreet/Cobrand/Lincolnshire.pm.html - seems bit easier to read rather than a series of headings and short descriptions when describing these functions.